### ISSUE TITLE
The method 'BehaviorSubject' isn't defined for the type 'LrsBloc'.  

### ISSUE THREAD

**Issue by swatantra1109** [jblough/TinCanDart#1]
Try correcting the name to the name of an existing method, or defining a method named 'BehaviorSubject'.

**Comment by jblough** [comment]
I added a specific version of the RxDart library to the pubspec.yaml files in the examples.  Does that resolve the issue when you build?

**Comment by swatantra1109** [comment]
Yes, Now I am getting 
Xcode's output:
↳
Error detected in pubspec.yaml:
No file or variants found for asset: assets/lrs.properties.


Command PhaseScriptExecution failed with a nonzero exit code
note: Using new build system
note: Building targets in parallel
note: Planning build
note: Constructing build description

**Comment by jblough** [comment]
If you look at the file [lrs.properties.template](https://github.com/jblough/TinCanDart/blob/c2a0ec57e16655f837dc4bf8465ab017dc40e148/test/lrs.properties.template), you can see the expected format of the lrs.properties file.  If you set up an account with a lrs provider (ScormCloud, YET Analytics, Veracity, Learning Locker, ...) you should be able to get the information to create a properties file.

**Comment by swatantra1109** [comment]
> If you look at the file [lrs.properties.template](https://github.com/jblough/TinCanDart/blob/c2a0ec57e16655f837dc4bf8465ab017dc40e148/test/lrs.properties.template), you can see the expected format of the lrs.properties file. If you set up an account with a lrs provider (ScormCloud, YET Analytics, Veracity, Learning Locker, ...) you should be able to get the information to create a properties file.

Really Appreciate the suggestion will do the same. Can We connect for xapi Need more clarity on that.

**Comment by jblough** [comment]
Yes, I believe https://github.com/jblough/TinCanDart#sample-usage is how you connect and submit an xAPI statement to the LRS
